src/controllers/API.js

```diff
@@ -185,9 +185,24 @@ class APIController {
     });
   }
 
-  static responseFromExternalError(error, requestAccepts) {
+  /**
+   * Builds a response from errors. Allows errors that occur outside of the
+   * library to be handled and returned in JSON API-compiant fashion.
+   *
+   * @param {Error|APIError|Error[]|APIError[]} errors Error or array of errors
+   * @param {string} requestAccepts Request's Accepts header
+   */
+  static responseFromExternalError(errors, requestAccepts) {
     let response = new Response();
-    response.errors = [APIError.fromError(error)];
+
+    // Convert to array
+    response.errors = Array.isArray(errors) ? errors : [errors];
+
+    // Convert Errors to APIErrors
+    response.errors = response.errors.map((error) => {
+      return error instanceof APIError ? error : APIError.fromError(error);
```

Comment:
No need for the `instanceof` check, since `APIError.fromError` handles that. Therefore, this can just be `response.errors = response.errors.map(APIError.fromError)`. 

Otherwise, this looks perfect!